html-forms/index.html

```diff
@@ -22,20 +21,18 @@ <h1>Coding Form!</h1>
     </div>
     <p>Favorite Language (choose one)</p>
     <div>
-      <input type="radio" name="language" value="HTML" checked="checked" /> HTML
-      <input type="radio" name="language" value="CSS" checked="checked" /> CSS
-      <input type="radio" name="language" value="JavaScript" checked="checked"> JavaScript
+      <input type="radio" name="language" value="HTML" checked="checked" /><label>HTML</label>
+      <input type="radio" name="language" value="CSS" checked="checked" /><label>CSS</label>
+      <input type="radio" name="language" value="JavaScript" checked="checked"><label>JavaScript</label>
```

Comment:
These labels still need to be properly configured to match with their paired `input`